### PR TITLE
Add check for vignette presence

### LINKED ISSUE #45: New check for vignettes
Because Dev Guide [clearly requires them](https://devguide.ropensci.org/building.html#documentation), as discussed in [this `software-review-meta` issue](https://github.com/ropensci/software-review-meta/issues/55) (link thanks to @maelle ).

### LINKED ISSUE #50: Clarify "add new" comment
In 10 new checks, will there be 10 checks between "add new" and the "end" comment? Or should new checks/summaries be added _right before_ such a comment?

### PR DESCRIPTION
**Pull request by maelle**
Fix #45
Fix #50

### CODE REVIEW

File: R/format-checks.R [hunk 1]
```diff
@@ -63,11 +63,13 @@ checks_to_markdown <- function (checks, render = FALSE) {
                  "</details>")
 
     # ----- Add new checks here -----
-    # Construct a separate function near the end of this file to convert the
```
Comment (maelle) [inline R/format-checks.R:66]: this is no longer true

File: R/pkgcheck-fn.R [hunk 2]
```diff
@@ -30,10 +30,13 @@ pkgcheck <- function (path) {
 
     out$gp <- pkgchk_gp_report (path)
 
-    # ----- Add new checks here -----
+    out$scrap <- pkgchk_has_scrap (path)
```
Comment (maelle) [inline R/pkgcheck-fn.R:33]: see #50

**Comment by maelle** [comment]
I'm not sure I understand the CI failures :grimacing:

**Comment by mpadge** [comment]
> I'm not sure I understand the CI failures :grimacing:

They're all fine - the `R-CMD-OS-check` is a windows issue that i'm pretty sure is not related to this package, and don't know how to fix. The `push-to-gitlab` just fails on branches, and can be ignored. It was just a test to set up that workflow.

**Review by mpadge — APPROVED** [review]
Looks great, thank you so much!

**Comment by maelle** [comment]
Why not deactivate the push-to-gitlab for branches then? :wink:

Thanks for merging!

**Comment by mpadge** [comment]
I had that thought too, and shall do right now. Thanks

**Comment by maelle** [comment]
@mpadge I made a PR!

**Comment by mpadge** [comment]
I already apologised over on #59, but happily repeat that here: Sorry!! didn't see it before i did my one

**Pull request merged**